source/index.rst

```diff
@@ -27,6 +27,7 @@ Tutorials
    manipulation
    perception
    calibration
+   torso_calibration
```

Comment:
let's actually put this as a subsection within the calibration tutorial -- I don't think we need to make this be a top-level menu item